examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -59,102 +79,260 @@ class TravelDestinationItem extends StatelessWidget {
+        : Ink.image(image: AssetImage(destination.assetName, package: destination.assetPackage), fit: BoxFit.cover, child: Container(),);
+
+    final List<Widget> children = [
+      // photo and title
+      SizedBox(
+        height: 184.0,
+        child: Stack(
+          children: <Widget>[
+            Positioned.fill(
+              child: image,
+            ),
+            Positioned(
+              bottom: 16.0,
+              left: 16.0,
+              right: 16.0,
+              child: FittedBox(
+                fit: BoxFit.scaleDown,
+                alignment: Alignment.centerLeft,
+                child: Text(destination.title,
+                  style: titleStyle,
                 ),
               ),
-              // share, explore buttons
-              ButtonTheme.bar(
-                child: ButtonBar(
-                  alignment: MainAxisAlignment.start,
-                  children: <Widget>[
-                    FlatButton(
-                      child: Text('SHARE', semanticsLabel: 'Share ${destination.title}'),
-                      textColor: Colors.amber.shade500,
-                      onPressed: () { /* do nothing */ },
-                    ),
-                    FlatButton(
-                      child: Text('EXPLORE', semanticsLabel: 'Explore ${destination.title}'),
-                      textColor: Colors.amber.shade500,
-                      onPressed: () { /* do nothing */ },
-                    ),
-                  ],
+            ),
+          ],
+        ),
+      ),
+      // description and share/explore buttons
+      Expanded(
+        child: Padding(
+          padding: const EdgeInsets.fromLTRB(16.0, 16.0, 16.0, 0.0),
+          child: DefaultTextStyle(
+            softWrap: false,
+            overflow: TextOverflow.ellipsis,
+            style: descriptionStyle,
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                // three line description
+                Padding(
+                  padding: const EdgeInsets.only(bottom: 8.0),
+                  child: Text(
+                    destination.description[0],
+                    style: descriptionStyle.copyWith(color: Colors.black54),
+                  ),
                 ),
+                Text(destination.description[1]),
+                Text(destination.description[2]),
+              ],
+            ),
+          ),
+        ),
+      ),
+    ];
+
+    if (destination.type == CardDemoType.static) {
+      children.add(
+        // share, explore buttons
+        ButtonTheme.bar(
+          child: ButtonBar(
+            alignment: MainAxisAlignment.start,
+            children: <Widget>[
+              FlatButton(
+                child: Text('SHARE', semanticsLabel: 'Share ${destination.title}'),
+                textColor: Colors.amber.shade500,
+                onPressed: () { /* do nothing */ },
```

Comment:
Someone told me once that the preferred behavior for catalog buttons is

`onPressed: () { print('pressed') },`

Reply:
done